documentation/release_notes_7.markdown

```diff
@@ -17,7 +17,7 @@ canonical: "/puppetdb/latest/release_notes.html"
 
 ## PuppetDB 7.11.1
 
-Released August 11 2022
+Released September 14 2022
```

Comment:
Suppose maybe this doesn't matter, but I'm not sure?

Reply:
These were legit updates.  I think it is ok.